hcert_spec.md

```diff
@@ -86,48 +89,57 @@ The Claim Key 1 is used to identify this claim.
 
 The Expiration Time (**exp**) claim SHALL hold a timestamp in the NumericDate format (as specified in [RFC 8392](https://tools.ietf.org/html/rfc8392) section 2) indicating for how long this particular signature over the Payload SHALL be considered valid, after which a Verifier MUST reject the Payload as expired. The purpose of the expiry parameter is to force a limit of the validity period of the EHC. The Claim Key 4 is used to identify this claim.
 
+The Expiration Time MUST not exceed the validity time in the DSC.  The verifier SHOULD check this.
```

Comment:
This somewhat contradicts the trust model which states that if a DSC is on the list, it is trusted. Otherwise it is not. If this is a validation rule on top of that, it could mean that a DSC which is on the list is suddenly considered untrusted.

An error by a HCERT issuer in this area could be more or less catastrophic. Envision the case that someone generating that DSC now (spring 2021) sets its validity time to two years (because they think the pandemic must surely be over by then), but the eHealth authority continues issuing certificates up to the expiry of the certificate. All HCERTs currently issued and valid would suddenly go invalid.

Reply:
So we should clarify this. Will edit.

Reply:
Raised issue #4